mkeditorpacks/mkeditorpacks

```diff
@@ -1,13 +1,38 @@
-#! /usr/bin/env python3
+#!/usr/bin/env python3
 #-*- coding: UTF-8 -*-
 
+# Copyright (c) 2014-2017, Daemon Developers
```

Comment:
I never understood what the date is meaning on this scope. Copyright expiration is based on time after author death, not based on time after the file was created or last touched.